operators/endpointmetrics/controllers/status/status_controller.go

```diff
@@ -7,15 +7,16 @@ import (
 	"os"
 	"reflect"
 
+	"github.com/stolostron/multicluster-observability-operator/operators/endpointmetrics/pkg/util"
```

Comment:
don't make this change.
to organize package from stolostron org in a separate section is the code style all obs codes are using.